Example pipelines/simple_pipeline.m

```diff
@@ -0,0 +1,220 @@
+% 
+% The computation in Homer3 is as follows:
+
+dod = hmrR_Intensity2OD(intensity_nirs)
+% Motion artefacts detection 
+% Parts of the data can be contaminated by artefacts, which could be systemic 
+% artefacts (e.g. heartbeat,respiration), or be caused by extrinsic factors (e.g. 
+% subject movement) (<https://www.spiedigitallibrary.org/journals/neurophotonics/volume-3/issue-3/031405/False-positives-and-false-negatives-in-functional-near-infrared-spectroscopy/10.1117/1.NPh.3.3.031405.full?SSO=1 
+% Tachtsidis&Scholkmann, 2016>). Both these types of artefacts need to be handled 
+% separately. In this tutorial, we focus on the latter.
+% 
+% Motion artefacts detection is often done by visual inspection of non-physiological 
+% data patterns or automated analysis. Homer3 provides the hmrR_MotionArtifactByChannel 
+% function, which detects outliers using standard deviation and amplitude thresholds. 
+% 
+% We will set these thresholds to be 50 and 5, respectively, and choose a 1 
+% second sliding window in which the channel-wise amplitude and standard deviation 
+% will be evaluated to detect outliers. If any active data channel exhibits a 
+% signal change greater than STDEVthresh or AMPthresh, then a segment of data 
+% around that time point is marked as a motion artifact. The channel-wise motion 
+% artifacts are recorded in the output matrix _tIncAutoCh, _where 1s represent 
+% motion-free data and 0s represent motion contaminated data. Time points in the 
+% recording where at least one channel has a motion artifact are indicated in 
+% a similar manner by the vector _tIncAuto_.
+
+nChannels = size(dod.dataTimeSeries, 2);
+% PARAMETERS:
+tMotion = 0.5;
+tMask = 1.0;
+STDEVthresh = 50;
+AMPthresh = 5.00;
+[tIncAuto, tIncAutoCh] = hmrR_MotionArtifactByChannel(dod, acquired.probe, [], [], [], ...
+    tMotion, tMask, STDEVthresh, AMPthresh);
+% Motion artefacts correction
+% Having already detected the motion artifacts in our data, we will now proceed 
+% to artefact handling. Artefact handling can be done in two ways: either, the 
+% data is removed from further analysis, or it is interpolated e.g. using cubic 
+% spline interpolation algorithm (<https://pubmed.ncbi.nlm.nih.gov/20308772/ Scholkmann 
+% et al., 2010>). We opted for the latter, which is imlpemented in homer as follows:
+
+dodMC = hmrR_MotionCorrectSpline(dod.copy, [], tIncAutoCh, 0.99); %Motion corrected ODs
+%% 
+% We now plot the resulting motion-corrected signals.
+
+t = dod.time;
+y = dod.dataTimeSeries;
+
+yMC = dodMC.dataTimeSeries;
+tstart = acquired.stim(1,2).data(1);
+
+% channels to plot 
+channels = 1:size(y, 2);
+figure; 
+ax1 = subplot(211); hold on
+for iCh = 1:length(channels)
+    plot(t, detrend(y(:, iCh)));
+    xlabel('time (s)')
+end
+title('Original raw ODs')
+ax2 = subplot(212); hold on
+for iCh = 1:length(channels)
+    plot(t, detrend(yMC(:, iCh)));
+    xlabel('time (s)')
+end
+title('Motion corrected ODs')
+linkaxes([ax1, ax2])
```

Comment:
This is for the future, not the current PR:

I wonder if [`tiledLayout `](https://es.mathworks.com/help/matlab/ref/tiledlayout.html) could be an alternative to this older approach of linking axes?